app/models/top_content.rb

```diff
@@ -4,6 +4,22 @@ class TopContent < ApplicationRecord
   # validate  :picture_size
   validate :url_size
 
+  def no_start_date
```

Comment:
booleanを返すメソッドの場合は `?` をメソッド名につけましょう（他のメソッドも同様）
```suggestion
  def no_start_date?
```